doc/source/whatsnew/v1.3.0.rst

```diff
@@ -110,6 +110,47 @@ both XPath 1.0 and XSLT 1.0 is available. (:issue:`27554`)
 
 For more, see :ref:`io.xml` in the user guide on IO tools.
 
+|AsyncEngine|_ and |AsyncConnection|_ now supported in Pandas SQL Functions/Methods
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
you would want to copy this to io.rst for posterity, ok to have this example as well